04_getting_started_js/containsAll.js

```diff
@@ -0,0 +1,14 @@
+const containsAll = function(elements, arr) {
+  const allFound = elements.every( (elementsEl) => {
+    const helpArr = arr.filter((arrEl) => arrEl === elementsEl);
```

Comment:
There are better ways to write this function. Hint: Try to search for other built in functions.